src/main/java/nextstep/security/authorization/AuthorityAuthorizationManager.java

```diff
@@ -3,32 +3,40 @@
 import jakarta.servlet.http.HttpServletRequest;
 import nextstep.security.authentication.Authentication;
 
-import java.util.Collections;
-import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
 public class AuthorityAuthorizationManager implements AuthorizationManager<HttpServletRequest> {
-    private final Set<String> defined_authorities = new HashSet<>();
+    private final RoleHierarchy defined_authorities;
```

Comment:
```suggestion
    private final RoleHierarchy definedAuthorities;
```

해당 변수만 스네이크 케이스로 작성되어 있네요!